<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageMarkupDialog</class>
 <widget class="QDialog" name="ImageMarkupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Image</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Image Address:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="E5PathPicker" name="imagePicker" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Enter the image path or URL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Alternative Text:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="altTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="sizeCheckBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Keep Original Size</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="aspectRatioCheckBox">
     <property name="text">
      <string>Keep Aspect Ratio</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="widthSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="heightSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>imagePicker</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>altTextEdit</tabstop>
  <tabstop>sizeCheckBox</tabstop>
  <tabstop>aspectRatioCheckBox</tabstop>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageMarkupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageMarkupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>aspectRatioCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widthSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>heightSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
